release-notes/opensearch-knn.release-notes-1.0.0.0.md

```diff
@@ -1,17 +1,38 @@
 ## Version 1.0.0.0 Release Notes
 
 Compatible with OpenSearch 1.0.0
+
+### Features
+
+* Add support for L-inf distance in AKNN, custom scoring and painless scripting ([#315](https://github.com/opendistro-for-elasticsearch/k-NN/pull/315))
```

Comment:
I believe inter-product space was added as well.

Reply:
You are right. It is interesting for some reason it was not picked up by our release drafter. I added it manually by looking over all merged PR.